app/LinkLiteManager/Services/RquestPollingService.cs

```diff
@@ -0,0 +1,155 @@
+using System.Text;
+using LinkLiteManager.Dto;
+using LinkLiteManager.OptionsModels;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+using RabbitMQ.Client;
+
+namespace LinkLiteManager.Services;
+
+internal interface IRquestPollingService
+{
+  Task PollRquest(CancellationToken stoppingToken);
+}
+public class RquestPollingService: IRquestPollingService
+{
+  private readonly RquestConnectorApiClient _rquestApi;
+  private readonly ILogger<RquestPollingService> _logger;
+  private readonly RquestPollingServiceOptions _config;
+  private Timer? _timer;
+  private int executionCount = 0;
+  
+  public RquestPollingService(
+    RquestConnectorApiClient rquestApi,
+    ILogger<RquestPollingService> logger,
+    IOptions<RquestPollingServiceOptions> config)
+  {
+    _logger = logger;
+    _rquestApi = rquestApi;
+    _config = config.Value;
+  }
+  public async Task PollRquest(CancellationToken stoppingToken)
+  {
+    while (!stoppingToken.IsCancellationRequested)
+    {
+      executionCount++;
+
+      _logger.LogInformation(
+        "Scoped Processing Service is working. Count: {Count}", executionCount);
```

Comment:
♻️ Not sure we need these boilerplate info logs, here and `47` and possible in the hosted service too?

Or at least if we do want to keep them, make them `LogDebug()` instead of info and put the actual service and/or method name in (you can use `nameof(RquestPollingService)` for example)